src/exchange_wrappers/TestExchangeWrapper.ts

```diff
@@ -0,0 +1,40 @@
+import web3Utils from 'web3-utils';
+import BN from 'bn.js';
+import BigNumber from 'bignumber.js';
+import { TestOrder } from '../types';
+import { TestExchangeWrapper as Contract } from '../../migrations/deployed.json';
+
+export class TestExchangeWrapper {
```

Comment:
You need to actually instances of these to `OrderMapper.ts`, and copy everything else that's done there